### ISSUE TITLE
Uses Google, not my specified search engine

### ISSUE THREAD

**Issue by marv3lls** [mozilla/firefox-voice#1822]
Pretty straight forward? I don't want to give them ANY data I don't have to, that's a big part of why I use Firefox! When I do a thing like say "Open DuckDuckGo", it redirects through *Google* for some reason!

**Comment by marv3lls** [comment]
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/202692/88016965-7c478f80-cad9-11ea-861e-2abee3373545.png">

**Comment by awallin** [comment]
@marv3lls Firefox Voice should respect the default search engine you've selected in the browsers preference. Do you have DuckDuckGo set as your default search engine?

**Comment by Vinnl** [comment]
@awallin I have the same issue and yes, it's my default search engine:

![search-engine](https://user-images.githubusercontent.com/4251/88635675-39c51a80-d0b8-11ea-9d37-1f90937d1c70.gif)

**Comment by marv3lls** [comment]
Yes @awallin , @Vinnl 's comment shows basically the same as my setup.

**Comment by awallin** [comment]
@marv3lls @Vinnl to clarify when you do a voice search such as _"Search for sushi recopies"_ the results you see are from your default search engine (DuckDuckGo) but when you navigate using a phrase such as _"Go to Mozilla"_ Google is acting as the referral and you'd prefer your default search engine be used in that instance?

**Comment by Vinnl** [comment]
That phrase goes to Google as well, but doesn't execute a search:

![search](https://user-images.githubusercontent.com/4251/88956286-730aaf80-d29d-11ea-8b37-09730a5ac06b.gif)

**Comment by Uffje** [comment]
Yes I have the same issue as Vinnl
